testsupport/tiers/checks.go

```diff
@@ -529,7 +529,7 @@ type appstudiolargeTierChecks struct {
 
 func (a *appstudiolargeTierChecks) GetClusterObjectChecks() []clusterObjectsCheck {
 	return clusterObjectsChecks(
-		clusterResourceQuotaDeploymentCount("600", "100"),
+		clusterResourceQuotaDeploymentCount("600", "100", "2"),
```

Comment:
It should be added to the base1ns tier check and all tiers which base on that tier. Not the appstudiolarge tier check. Right?

Reply:
ah, missed that this was an appstudio tier.